packages/plans-grid/src/plans-feature-list/index.tsx

```diff
@@ -27,60 +28,94 @@ const ChevronDown = (
 	</svg>
 );
 
+const DomainFeatureAnnualDiscountNudge: React.FunctionComponent< {
+	billingInterval: BillingIntervalType;
+	__: typeof import('@wordpress/i18n').__;
+} > = ( { billingInterval, __ } ) => (
+	<span
+		className="plans-feature-list__domain-summary-nudge"
+		aria-label={
+			billingInterval === 'ANNUALLY'
+				? __( 'Included with annual plans', __i18n_text_domain__ )
+				: __( 'Only included with annual plans', __i18n_text_domain__ )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Included with annual plans' )` *ES Score: 9*